app-crypt/tpm2-pkcs11/tpm2-pkcs11-1.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1
+
+DESCRIPTION="A PKCS#11 interface for TPM2 hardware"
+HOMEPAGE="https://tpm2-software.github.io/"
+SRC_URI="https://github.com/tpm2-software/tpm2-pkcs11/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+# Test requires the IBM TPM simulator that's not in portage
+RESTRICT="test"
+
+RDEPEND="app-crypt/p11-kit
+	>=app-crypt/tpm2-abrmd-2.1.0
```

Comment:
Always satisfied in ::gentoo tree, version restriction can be lifted off.

Reply:
Done